src/plugins/inputs/index.js

```diff
@@ -47,7 +45,6 @@ const getCommands = () =>
 	Array.from(userInputs.keys())
 		.concat(Array.from(pluginCommands.keys()))
 		.map((command) => `/${command}`)
-		.concat(clientSideCommands)
```

Comment:
I like that we're not sending the client the commands it already knows now :).

Reply:
This is now done in `client/js/socket-events/commands.js`, but thats a change from the previous PR and not strictly necessary.